1.segment_images/scripts/2.make_nuclei_segmentation_videos.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+# coding: utf-8
+
```

Comment:
Recommend adding a title for this file/notebook. The file name says videos but is this making GIFs? Recommend being more explicit.

Reply:
added